package.json

```diff
@@ -77,7 +77,8 @@
     "commands": [
       {
         "command": "extension.fabric8AnalyticsWidgetFullStack",
-        "title": "Dependency Analytics Report ..."
+        "title": "Dependency Analytics Report...",
+        "category": "Dependency Analytics"
```

Comment:
Now your command title shouldn't have "Dependency Analytics", it will be redundant.

Reply:
ack